spec/tapjoy/ldap/group/add_user_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../../../spec_helper'
```

Comment:
you can save some sanity by just doing require 'spec_helper' and require 'user' etc. in these. require should recurse outwards until it hits the file.